elasticsearch/run.sh

```diff
@@ -24,6 +27,31 @@ retry=$RETRY_COUNT
 max_time=$(( RETRY_COUNT * RETRY_INTERVAL ))	# should be integer
 timeouted=false
 
+# modify configs as needed
+mkdir -p ${HOME}/config
+cp ${ES_CONF}/* ${HOME}/config
+pushd ${HOME}/config
+  if [ -f ${HOME}/config/logging.yml ] && ! grep -q "io\.openshift\.log4jextras" ${HOME}/config/logging.yml ; then
+    python -c '
+import yaml
+with open("logging.yml") as file:
+  c = yaml.load(file)
+  for appender in ["file", "deprecation_log_file", "index_search_slow_log_file", "index_indexing_slow_log_file"]:
+    if c["appender"][appender]:
+      f = c["appender"][appender]
+      f["type"] = "io.openshift.log4jextras.appender.DailyRollingFileAppender"
+      f["maxBackupIndex"] = "${ES_LOGS_MAX_BACKUP}"
+      f["datePattern"] = "${ES_LOGS_DATE_PATTERN}"
```

Comment:
```
      f["maxBackupIndex"] = sys.argv[1]
      f["datePattern"] = sys.argv[2]
```

Reply:
with the construct of `f["maxBackupIndex"] = "${ES_LOGS_MAX_BACKUP}"` elastic will evaluate environment vars which we use as a pattern before this change

Reply:
ah, ok